.github/workflows/main.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     strategy:
       matrix:
         # Specify what versions of php you want to test
-        php-versions: ['7.4', '8.2']
+        php-versions: ['7.4', '8.3']
```

Comment:
Maybe need 7.4 + 8.1 + 8.3 ?

Reply:
It seemed to me that the idea here is to have the minimum & maximum version runs, so I just replaced the 8.2 with 8.3 as  minimal change here.

Reply:
I don't know … unsure.

But never mind : if it works on both : it surely works on middle.